driver-core/src/main/java/com/datastax/driver/core/Host.java

```diff
@@ -221,6 +241,32 @@ public VersionNumber getCassandraVersion() {
     }
 
     /**
+     * The DSE version the host is running.
+     * <p/>
+     * It is also possible for this information to be unavailable. In that case
+     * this method returns {@code null}, and the caller should always be aware of this
+     * possibility.
+     *
+     * @return the DSE version the host is running.
+     */
+    public VersionNumber getDseVersion() {
```

Comment:
So after doing a little more testing, it seems that newer DSE versions handles this much better.  On 4.8.3 I saw that a Solr workload never sets the dse_version column, where it works just fine.